app/server/utils/ussd/kenya_ussd_state_machine.py

```diff
@@ -310,6 +340,17 @@ def get_select_transfer_usage(self, user_input):
         selected_tranfer_usage_id = self.session.get_data('transfer_usage_mapping')[idx]['id']
         return TransferUsage.query.get(selected_tranfer_usage_id)
 
+    def is_ussd_signup(self, user_input):
+        return self.user.registration_method == RegistrationMethod.USSD_SIGNUP.value
+
+    def process_account_creation_request(self, user_input):
+        try:
+            attach_transfer_account_to_user(self.user)
```

Comment:
please advise in which cases can this error happen? can we add some metadata that can be passed to support?

Reply:
in the function `attach_transfer_account_to_user` there's a blockchain task that is responsible for creating a wallet, that's what I anticipate maybe result in an error.

Reply:
> that's what I anticipate maybe result in an error.

what would the error mean, though? would it be that the tasker doesn't take the task? or would it be an error from the blockchain attempt? If only the former, how will we catch an error with the latter?